spec/index.md

```diff
@@ -4723,15 +4847,6 @@ if A.settings.controllers is not null:
   New_controllers = A.settings.controllers
 else:
   New_controllers = [M.caller]
-
-if New_compute_allocation > 0 or New_memory_allocation > 0:
```

Comment:
same comment as for canister creation applies here: it used to be that the implementation would only check the freezing limit under this if-condition

Reply:
https://github.com/dfinity/ic/blob/b53c7fff38f7e5adc324283b162abd86bf356d4e/rs/execution_environment/src/execution/install_code.rs#L230